tools/common/CompilerFlags.cs

```diff
@@ -16,7 +17,7 @@ public class CompilerFlags
 		public HashSet<string> WeakFrameworks;
 		public HashSet<string> LinkWithLibraries; // X, added to Inputs
 		public HashSet<string> ForceLoadLibraries; // -force_load X, added to Inputs
-		public HashSet<string> OtherFlags; // X
+		public HashSet<string[]> OtherFlags; // X
```

Comment:
that makes querying flags nearly impossible :|

Reply:
Can you explain? I don't quite understand what you mean here.

Reply:
The problem is that the order matters... if a place wants to add '-L' and '/path/to/foo', those arguments have to be kept in the same order, and a HashSet doesn't preserve order.

Reply:
yeah, you can't query it without having re-creating the same array
not a big deal in the current code, just that I like `HashSet` for reasons that an array destroy :)
I don't have anything better to suggest so...